src/symbol/observable.ts

```diff
@@ -0,0 +1,23 @@
+import {root} from '../util/root';
+
+let $$observable: any;
+
+export function getSymbolObservable() {
+  if ($$observable) {
+    return $$observable;
+  }
+  let Symbol = root.Symbol;
+
+  if (typeof Symbol === 'function') {
+    if (Symbol.observable) {
+      $$observable = Symbol.observable;
+    } else {
+        $$observable = Symbol('observable');
+        Symbol.observable = $$observable;
+    }
+  } else {
+    $$observable = '@@observable';
+  }
+
+  return $$observable;
+}
```

Comment:
can we have this just return the `$$observable` from the module? That way it's cached and we don't need to call `getSymbolObservable()` more than once.

I'd prefer that we had two named exports:  `getSymbolObservable(root)` (so we can test it with different root values), and `$$observable` which is really just `getSymbolObservable(root)` having been called and exported.